### PR TITLE
Add support for feature flags to ProjectService.

### PR DESCRIPTION
**Pull request by jumarko**

### CODE REVIEW

**Comment by martiner** [comment]
- [ ] please replace `AbstractGoodDataIT.readResource` calls with `ResourceUtils`

**Comment by martiner** [comment]
- [ ] add acceptance test for feature flags

File: src/main/java/com/gooddata/GoodData.java [hunk 1]
```diff
@@ -52,9 +52,9 @@
 
     public static final String GDC_REQUEST_ID_HEADER = "X-GDC-REQUEST";
 
-    private static final String PROTOCOL = "https";
-    private static final int PORT = 443;
-    private static final String HOSTNAME = "secure.gooddata.com";
+    protected static final String PROTOCOL = "https";
```
Comment (pirkogdc) [inline src/main/java/com/gooddata/GoodData.java]: do u have any use case for exposing this?

Comment (jumarko) [inline src/main/java/com/gooddata/GoodData.java]: Of course, I have - anytime you need to extend GoodData java you are forced to copy the default values: https://github.com/gooddata/msf-tools/blob/perf-tests/dl-tool/src/main/java/com/gooddata/msf/dltool/service/DlGoodData.java#L19

Comment (martiner) [inline src/main/java/com/gooddata/GoodData.java]: I cherry-picked this commit to master.

**Comment by martiner** [comment]
please rebase

**Comment by jumarko** [comment]
Closing. I'll make another pull request from different branch.

**Pull request closed without merging**